assets/css/error-rescue.css

```diff
@@ -240,18 +245,37 @@
   margin: 0;
   background-color: var(--line-ins-background);
 }
-.kite-error-rescue-sidebar a.icon-info {
+
+.vscode-high-contrast .kite-error-rescue-sidebar .recent del,
```

Comment:
Imo the best practice is to extract every theme-related thing into variables defined in each theme files so that each "feature" stylesheet doesn't have to care about it (that was the purpose of using CSS variables in the first place).